_unittest/test_26_emit.py

```diff
@@ -559,17 +565,24 @@ def test_analyze_manually(self):
                     subfolder = os.path.join(f.path, "EmitDesign1")
             file = max([f for f in os.scandir(subfolder)], key=lambda x: x.stat().st_mtime)
             self.aedtapp._emit_api.load_project(file.path)
-            assert self.aedtapp.results.result_loaded
-            assert Revision(self.aedtapp, "Revision 1") is not None
-            domain = self.aedtapp.interaction_domain()
+            assert rev.revision_loaded
+            domain = self.aedtapp.results.interaction_domain()
             assert domain is not None
             engine = self.aedtapp._emit_api.get_engine()
             assert engine is not None
             assert engine.is_domain_valid(domain)
+            assert rev.is_domain_valid(domain)
             interaction = engine.run(domain)
             assert interaction is not None
+            interaction2 = rev.run(domain)
+            assert interaction2 is not None
+            assert interaction2.is_valid()
+            self.aedtapp.results.delete_revision(rev.name)
+            assert not interaction.is_valid()
```

Comment:
In this case, is it true that interaction and interaction2 both should not be valid? Should we add assertion for interaction2 ?